sa/sa_test.go

```diff
@@ -3268,24 +2601,8 @@ func TestGetValidAuthorizations2(t *testing.T) {
 	sa, fc, cleanUp := initSA(t)
 	defer cleanUp()
 
-	// Create a valid old style authorization and a valid
-	// new style authorization
+	// Create a valid new style authorization
```

Comment:
```suggestion
	// Create a valid authorization
```

tiny nit: I think "new style" is confusing when there's only one style implemented.